src/app/modules/course-overview/course-overview.module.ts

```diff
@@ -0,0 +1,24 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+import { CourseOverviewComponent } from './components/course-overview/course-overview.component';
+import { MatCardModule } from '@angular/material/card';
+import { MatButtonToggleModule } from '@angular/material/button-toggle';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import { SharedModule } from 'src/app/shared/shared.module';
+import { CourseCardComponent } from './components/course-card/course-card.component';
+
+@NgModule({
+  declarations: [CourseOverviewComponent, CourseCardComponent],
+  imports: [
+    CommonModule,
+    RouterModule,
+    MatButtonModule,
```

Comment:
material module

Reply:
I didn't find the material module?

Reply:
that's what I meant. sry. It's really good to separate out the material deps since as our app grows we will be importing more things